tools/pd-ctl/pdctl/command/store_command.go

```diff
@@ -228,12 +230,25 @@ func storeLimitSceneCommandFunc(cmd *cobra.Command, args []string) {
 
 func showStoreCommandFunc(cmd *cobra.Command, args []string) {
 	prefix := storesPrefix
+	if len(args) > 1 {
+		cmd.Usage()
+		return
+	}
 	if len(args) == 1 {
 		if _, err := strconv.Atoi(args[0]); err != nil {
 			cmd.Println("store_id should be a number")
 			return
 		}
 		prefix = fmt.Sprintf(storePrefix, args[0])
+	} else {
+		if state := cmd.Flag("state"); state != nil && state.Value.String() != "" {
+			stateValue, ok := metapb.StoreState_value[state.Value.String()]
```

Comment:
Maybe we can support get multiple states?

Reply:
I will have a try.